docs/source/configure.rst

```diff
@@ -1,14 +1,7 @@
 Configuration
 =============
 
-
-The package contains a configuration file ``config.json``. After the package is installed you can check the initial
-configuration parameters using the command line::
-
-$ sentinelhub.config --show
-
-The same configuration can be seen by instantiating an instance of :class:`~sentinelhub.config.SHConfig`;
-without passing parameters the ``config.json`` will be used to populate the values:
+Part of the package configuration is represented by the :class:`~sentinelhub.config.SHConfig` class. It can be adjusted and passed to most of the functions and constructors of the ``sentinelhub`` package to either provide credentials or modify behavior such as the number download processes.
```

Comment:
```suggestion
Part of the package configuration is represented by the :class:`~sentinelhub.config.SHConfig` class. It can be adjusted and passed to most of the functions and constructors of the ``sentinelhub`` package to either provide credentials or modify behavior such as the number of download processes.
```